docs/user-guide/creating-application/app-metrics.md

```diff
@@ -1,14 +1,15 @@
 # Application Metrics
 
-Application metrics can be enabled to see your application's metrics.
+To get application metrics, [show application metrics](../creating-application/deployment-template.md#3.-show-application-metrics) option should be enabled from deployment template.
 
 ## Standard Metrics
 
-Devtron provides certain metrics (CPU and Memory utilization) for each application by default i.e. you do not need to enable “Application metrics”. However, prometheus needs to be present in the cluster and the endpoint of the same should be updated in Global Configurations --> Clusters & Environments section. 
+Devtron provides certain metrics (CPU and Memory utilization) for each application by default i.e. you do not need to enable the application metrics. However, prometheus needs to be present in the cluster and the endpoint of the same should be updated in Global Configurations --> [Clusters & Environments](../global-configurations/cluster-and-environments.md) section. 
```

Comment:
Use this [link](https://docs.devtron.ai/getting-started/global-configurations/cluster-and-environments#3.-prometheus-info) instead on cluster and env. 

**NOTE: Attached link will change due to the below pointer.**

Pre-requisite changes on cluster and env page:
1. Change Prometheus info title > Enabling application metrics for a cluster
2. Add a screenshot highlighting the metrics configurations section
3. If you want to see your application metrics against your applications deployed in Kubernetes, [Install Prometheus](https://devtron.ai/blog/setup-monitoring-stack/) in your Kubernetes cluster. Prometheus is a powerful solution to provide graphical insight into your application behavior. The below inputs are required to configure your Prometheus into Devtron’s tool.

Reply:
Will do this part in this (https://github.com/devtron-labs/devtron/pull/2367) PR.